doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -2997,7 +2996,7 @@ illustrated in the following example.
 
   .. coqtop:: none
 
-     Abort All.
+     Abort All. Reset Test.
```

Comment:
No, please, don't use Reset other than `Initial`

Reply:
OK but why?

Reply:
Sorry, it was a reflex. If this is for coqtop, then it is fine